skelebot/components/prime.py

```diff
@@ -34,4 +35,9 @@ def execute(self, config, args):
         built so as to be ready for deployment
         """
 
-        return docker.build(config)
+        status = docker.build(config)
+        if (status == 0) & (args.output is not None):
```

Comment:
I'm not sure, but I think if you define this `output` as a required argument, then it will never be `None`.
Example: https://docs.python.org/3/library/argparse.html#required

Reply:
It needs to be an optional arg. We want the possibility of not saving the image as an output file.